app/imports/startup/server/Mongo.js

```diff
@@ -31,3 +32,17 @@ if (Items.find().count() === 0) {
     Meteor.settings.defaultItems.map(data => addItems(data));
   }
 }
+
+/** Initialize the database with a default data document. */
+function addCategories(data) {
+  console.log(`  Adding: ${data.name} (${data.owner})`);
```

Comment:
This looks copy pasted from another file... Did you mean to print the name and owner in console?  I was thinking this would be more like:

console.log(`    Adding: ${data.name} (${data.group})`);

Reply:
o yes, you're right, that is my mistake, I'll fix it right now